src/main/java/org/prebid/server/bidder/bizzclick/BizzclickBidder.java

```diff
@@ -100,7 +103,11 @@ private static MultiMap headers(Device device) {
     }
 
     private String buildEndpointUrl(ExtImpBizzclick ext) {
-        return endpointUrl.replace(URL_SOURCE_ID_MACRO, HttpUtil.encodeUrl(ext.getPlacementId()))
+        final String host = StringUtils.isEmpty(ext.getHost()) ? DEFAULT_HOST : ext.getHost();
+        final String sourceId = StringUtils.isEmpty(ext.getSourceId()) ? ext.getPlacementId() : ext.getSourceId();
```

Comment:
Would be great to have tests for this logic.

Reply:
Done.